velox/serializers/tests/PrestoSerializerTest.cpp

```diff
@@ -339,6 +350,36 @@ TEST_P(PrestoSerializerTest, dictionaryWithExtraNulls) {
   testRoundTrip(dictionary);
 }
 
+// TODO: re-enable after we support dictionary encoding in Presto serializer.
+TEST_P(PrestoSerializerTest, DISABLED_dictionaryEncoding) {
+  const vector_size_t vectorSize = 1'000;
+  auto base = makeFlatVector<int64_t>(10, [](auto row) { return row; });
+
+  BufferPtr indices =
+      AlignedBuffer::allocate<vector_size_t>(vectorSize, pool_.get());
+  auto rawIndices = indices->asMutable<vector_size_t>();
+  for (auto i = 0; i < vectorSize; i++) {
+    rawIndices[i] = i % 10;
+  }
+
+  auto dictionary =
+      BaseVector::wrapInDictionary(nullptr, indices, vectorSize, base);
+  serializer::presto::PrestoVectorSerde::PrestoOptions options;
+  options.encodings = {VectorEncoding::Simple::DICTIONARY};
+  testRoundTrip(dictionary, &options);
+}
+
+// TODO: re-enable after we support constant encoding in Presto serializer.
+TEST_P(PrestoSerializerTest, DISABLED_constantEncoding) {
+  auto constantVector =
+      BaseVector::createConstant(INTEGER(), 100, 1024, pool_.get());
+  auto data = makeRowVector({constantVector});
+  serializer::presto::PrestoVectorSerde::PrestoOptions options;
+  options.encodings = {VectorEncoding::Simple::CONSTANT};
+  testRoundTrip(data, &options);
```

Comment:
Given you've already gone to the trouble of creating the RowVector, why not use testEncodedRoundTrip to avoid creating the PrestoOptions?